### ISSUE TITLE
Repeated warnings about parsing the coordinate system information

### ISSUE THREAD

**Issue by peanutfun** [CLIMADA-project/climada_python#486]
PRs #455 and #459 introduced a new warning when the coordinate system information cannot be parsed by `climada.util.plot.get_transformation` and the system falls back to the default `PlateCarree` transformation. This warning is now issued repeatedly even when using the default coordinate systems. What is more, different kinds of coordinate system information seems to be passed to the function `get_transformation`, which (to me) hints at a more severe underlying issue with what coordinate system information is passed. Consider this simple example:

```python
from climada.util.api_client import Client

client = Client()
litpop = client.get_litpop_default("BGD")
litpop.plot_raster()


```

It results in the warning
```
2022-06-21 10:12:39,743 - climada.util.plot - WARNING - Error parsing coordinate system 'GEOGCS["WGS 84",DATUM["WGS_1984",SPHEROID["WGS 84",6378137,298.257223563,AUTHORITY["EPSG","7030"]],AUTHORITY["EPSG","6326"]],PRIMEM["Greenwich",0],UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],AXIS["Latitude",NORTH],AXIS["Longitude",EAST],AUTHORITY["EPSG","4326"]]'. Using projection PlateCarree in plot.
```

The data here seems to be the "Well Known Text" information for EPSG:4326 / WGS 84. Why is this whole blob passed instead of a simple identifier?

Another example is creating the centroids from lat/lon information with the default CRS:
```python
import numpy as np
from scipy.sparse import csr_matrix
from climada.hazard import Hazard
from climada.hazard.centroids.centr import Centroids

latitude = np.linspace(21.0, 27.0, 10)
longitude = np.linspace(88.0, 93.0, 10)
lat, lon = np.meshgrid(latitude, longitude, indexing="ij")

hazard = Hazard()
hazard.centroids = Centroids.from_loat_lon(lat.flatten(), lon.flatten())  # Uses default CRS
hazard.centroids.set_lat_lon_to_meta()
hazard.intensity = csr_matrix(np.ones((1, lat.size)))
hazard.plot_intensity()
```

The warning then reads:
```
2022-06-21 10:15:58,210 - climada.util.plot - WARNING - Error parsing coordinate system 'EPSG:4326'. Using projection PlateCarree in plot.
```
[EPSG:4326](https://epsg.io/4326) seems to be a valid identifier for a coordinate system, so I don't understand why this could not be parsed.

**Comment by emanuel-schmid** [comment]
| [EPSG:4326](https://epsg.io/4326) seems to be a valid identifier for a coordinate system ...

help text of cartopy.crs.epsg explains this:
```text
epsg(code)
    Return the projection which corresponds to the given EPSG code.
    
    The EPSG code must correspond to a "projected coordinate system",
    so EPSG codes such as 4326 (WGS-84) which define a "geodetic coordinate
    system" will not work.
    
    Note
    ----
        The conversion is performed by pyproj.CRS.
```

**Comment by peanutfun** [comment]
Ok I don't really understand this detail 😬 Should we then silence the warning altogehter? I think in this context it's really confusing and a warning should not be issued if you cannot do anything to prevent it 🤔

**Comment by emanuel-schmid** [comment]
Apparently solved with #520
